dashboard/test/factories/factories.rb

```diff
@@ -244,6 +244,16 @@
       oauth_token_expiration 'fake-oauth-token-expiration'
     end
 
+    trait :unmigrated_untrusted_email_sso do
+      unmigrated_sso
+      after(:create) do |user|
+        if user.student?
+          user.hashed_email = nil
+          user.save!
```

Comment:
For Clever (at least) we don't record any email or hashed email anymore, because we don't trust the email they provide to actually belong to the user.